** Backup-managers version string includes "debian"
   We want people to know that it differs from upstream.

   Provided by: Alexis Sukrieh <sukria@sukria.net>


--- a/backup-manager
+++ b/backup-manager
@@ -27,7 +27,7 @@
 
 RELEASE="true"
 REVISION='$Revision: 590 $'
-VERSION="0.7.8"
+VERSION="0.7.8-debian2"
 
 # All the paths we provide
 libdir="/usr/share/backup-manager"
